zenoh/src/net/routing/queries.rs

```diff
@@ -1461,6 +1461,45 @@ fn compute_final_route(
     }
 }
 
+#[inline]
+fn compute_local_replies(
+    tables: &Tables,
+    prefix: &Arc<Resource>,
+    suffix: &str,
+    face: &Arc<FaceState>,
+) -> Vec<(WireExpr<'static>, ZBuf)> {
+    let mut result = vec![];
+    if face.whatami == WhatAmI::Client {
```

Comment:
It's not clear to me why this computation would apply only to clients.

Reply:
Added comment